src/Parser/DisciplinePriest/Modules/Features/AtonementAttribution.js

```diff
@@ -0,0 +1,30 @@
+import Module from 'Parser/Core/Module';
+import isAtonement from '../Core/isAtonement';
+
+class AtonementAttribution extends Module {
+
+  reorderEvents(events) {
+    // Push down atonement healing on self to the next atonement healing
+    let movesToMake = [];
+
+    for(let i = 0; i < events.length; i++){
```

Comment:
please change this to a forEach;

```jsx
events.forEach((event, index) => {
  [rest of your code]
});
```

Reply:
Also I think if you name `event` and `index` differently you might get more understandable code. These are the self-healing Atonements right? So how about `atonementSelfHeal` or `atonementSelfHealEvent`?